scss/custom.scss

```diff
@@ -128,6 +128,10 @@ ol {
     border-radius: 5px;
 }
 
+.nav-link {
+    color: #ffff
```

Comment:
Can we use the `$white` variable here instead?

Reply:
good idea. just implemented this